README.md

```diff
@@ -166,9 +166,24 @@ spec:
 | Private Builder Image Registry     | ⚪️     |      |    |
 | Runtime Base Image    | ⚪️    |      |    |
 | Binary builds         |       |      |    |
```

Comment:
I think there are some valuable lessons learned, especially in light of recent build v1 customer activity, as devex has discussed internally along with @bparees, that would should engage with @sbose78 's team on when they are ready to start on this.

Not sure if there is common library logic from buildv1 to share, especially since buildv2 may want to go down a different path than the tar to the build container path buildv1 employs  today.

@adambkaplan - initial reaction, this feels like consulting work on our end vs. R&D and/or provided code on our end, but at least warrants discussion on our end